<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerrainFormation</class>
 <widget class="QWidget" name="TerrainFormation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="picogen::qt4::StashFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="selectKind">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QFrame" name="easyKind">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="margin">
                <number>16</number>
               </property>
               <item>
                <widget class="QCommandLinkButton" name="heightmapCLB">
                 <property name="text">
                  <string>Heightmap</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/toplevel-navigation/resources/sketchy_cc-attribution-noncommercial_mountain-sketch.png</normaloff>:/toplevel-navigation/resources/sketchy_cc-attribution-noncommercial_mountain-sketch.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>92</width>
                   <height>64</height>
                  </size>
                 </property>
                 <property name="description">
                  <string>Use a grayscale-heightmap.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCommandLinkButton" name="importCLB">
                 <property name="text">
                  <string>Import</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/toplevel-navigation/resources/crystalclear_lgpl_folder.png</normaloff>:/toplevel-navigation/resources/crystalclear_lgpl_folder.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>92</width>
                   <height>64</height>
                  </size>
                 </property>
                 <property name="description">
                  <string>Re-use existing terrain.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCommandLinkButton" name="parametricPresetCLB">
                 <property name="text">
                  <string>Parametric Preset</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/toplevel-navigation/resources/bagsandboxes_freeforcommercial_preset.png</normaloff>:/toplevel-navigation/resources/bagsandboxes_freeforcommercial_preset.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>92</width>
                   <height>64</height>
                  </size>
                 </property>
                 <property name="description">
                  <string>Complex terrain ready to use!</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="toughKind">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="margin">
                <number>16</number>
               </property>
               <item>
                <widget class="QCommandLinkButton" name="quatschCodeEditorCLB">
                 <property name="text">
                  <string>Quatsch Code Editor</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/toplevel-navigation/resources/oxygen_gpl_quatschsourceeditor.png</normaloff>:/toplevel-navigation/resources/oxygen_gpl_quatschsourceeditor.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                 <property name="description">
                  <string>Reveal Turing-Completeness for your terrain!</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="quatschCodeEditorPage">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QuatschSourceEditor" name="quatschCodeEditor" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="quatschPresetEditorPage">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="picogen::qt4::QuatschPresetEditor" name="quatschPresetEditor" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>picogen::qt4::StashFrame</class>
   <extends>QFrame</extends>
   <header>stashframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>picogen::qt4::QuatschPresetEditor</class>
   <extends>QWidget</extends>
   <header>quatschpreseteditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QuatschSourceEditor</class>
   <extends>QWidget</extends>
   <header>quatschsourceeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
